airflow/gcp/operators/cloud_memorystore.py

```diff
@@ -0,0 +1,636 @@
+    :type location: str
+    :param instance_id: Required. The logical name of the Redis instance in the customer project with the
+        following restrictions:
+
+        -  Must contain only lowercase letters, numbers, and hyphens.
+        -  Must start with a letter.
+        -  Must be between 1-40 characters.
+        -  Must end with a number or a letter.
+        -  Must be unique within the customer project / location
+    :type instance_id: str
+    :param instance: Required. A Redis [Instance] resource
+
+        If a dict is provided, it must be of the same form as the protobuf message
+        :class:`~google.cloud.redis_v1.types.Instance`
+    :type instance: Union[Dict, google.cloud.redis_v1.types.Instance]
+    :param project_id: Project ID of the project that contains the instance. If set
+        to None or missing, the default project_id from the GCP connection is used.
+    :type project_id: str
+    :param retry: A retry object used to retry requests. If ``None`` is specified, requests will not be
+        retried.
+    :type retry: google.api_core.retry.Retry
+    :param timeout: The amount of time, in seconds, to wait for the request to complete. Note that if
+        ``retry`` is specified, the timeout applies to each individual attempt.
+    :type timeout: float
+    :param metadata: Additional metadata that is provided to the method.
+    :type metadata: Sequence[Tuple[str, str]]
+    :param gcp_conn_id: The connection ID to use connecting to Google Cloud Platform.
+    :type gcp_conn_id: str
+    """
+
+    template_fields = (
+        'location',
+        'instance_id',
+        'instance',
+        'project_id',
+        'retry',
+        'timeout',
+        'metadata',
+        'gcp_conn_id',
+    )
+
+    @apply_defaults
+    def __init__(
+        self,
+        location: str,
+        instance_id: str,
+        instance: Union[Dict, Instance],
+        project_id: str = None,
+        retry: Retry = None,
+        timeout: float = None,
+        metadata: Sequence[Tuple[str, str]] = None,
+        gcp_conn_id: str = "google_cloud_default",
+        *args,
+        **kwargs
+    ):
+        super().__init__(*args, **kwargs)
+        self.location = location
+        self.instance_id = instance_id
+        self.instance = instance
+        self.project_id = project_id
+        self.retry = retry
+        self.timeout = timeout
+        self.metadata = metadata
+        self.gcp_conn_id = gcp_conn_id
+
+    def execute(self, context: Dict):
+        hook = CloudMemorystoreHook(gcp_conn_id=self.gcp_conn_id)
+        result = hook.create_instance(
+            location=self.location,
+            instance_id=self.instance_id,
+            instance=self.instance,
+            project_id=self.project_id,
+            retry=self.retry,
+            timeout=self.timeout,
+            metadata=self.metadata,
+        )
+        return MessageToDict(result)
```

Comment:
I miss some nice logs here 😿 Meaningful logging is always helpful to see what's going on. WDYT @potiuk @TobKed

Reply:
Yes. I should add more logs.